src/collection/components/CollectionOrBundleOverview.tsx

```diff
@@ -364,14 +391,22 @@ const CollectionOrBundleOverview: FunctionComponent<CollectionOrBundleOverviewPr
 						{ id: collectionId }
 					);
 					break;
+
 				case 'createAssignment':
 					history.push(
 						generateAssignmentCreateLink('KIJK', `${collectionId}`, 'COLLECTIE')
 					);
 					break;
+
+				case 'createQuickLane':
+					setSelected(collectionId);
+					setIsQuickLaneModalOpen(true);
+					break;
+
 				case 'delete':
 					onClickDelete(collectionId);
 					break;
```

Comment:
Dit is dus het probleem met `onClick(whichButton: string) => void`-type componenten

Reply:
welk probleem bedoel je precies?

Reply:
mapping bloat, onnodige `switch/case`-statements, string-matching (`createQuickLane === ceateQuickLane`), ...